### PR TITLE
fixes RHOARDOC-1168: cleanup secured booster intro paragraph

### PR DESCRIPTION
**Pull request by rhoads-zach**
FYI @Ladicek

### CODE REVIEW

File: docs/topics/secured-booster-minishift.adoc [hunk 1]
```diff
@@ -1,7 +1,6 @@
 = Configuring Your {OpenShiftLocal}
-Before you can run the Secured Booster, you must update your {OpenShiftLocal} from the default configuration. You only need to do this once for the secured booster missions.
+Before you can run the Secured Booster, you must update your {OpenShiftLocal} from the default configuration. You only need to do this once for the secured booster mission.
```
Comment (Ladicek) [inline docs/topics/secured-booster-minishift.adoc]: Maybe "You only need to do this once for any of the secured boosters."? (Alternatively, "for any of the secured mission boosters") That would work well together with the following sentence.

Comment (Ladicek) [reply docs/topics/secured-booster-minishift.adoc]: (Frankly, in situation like this, I'm starting to hate the boosters/missions/whatever metaphor we use.)

Comment (rhoads-zach) [reply docs/topics/secured-booster-minishift.adoc]: I see what you mean. I updated based on your feedback. Thanks for the input.

**Comment by rhoads-zach** [comment]
@Ladicek fixed, let me know what you think.

File: docs/topics/secured-booster-minishift.adoc (hunk 1 above)
Comment (Ladicek) [reply docs/topics/secured-booster-minishift.adoc]: I don't see a change... :-)

Comment (rhoads-zach) [reply docs/topics/secured-booster-minishift.adoc]: @Ladicek sorry, i had commented before i pushed, it should be in there now

File: docs/topics/secured-booster-minishift.adoc [hunk 2]
```diff
@@ -1,7 +1,8 @@
 = Configuring Your {OpenShiftLocal}
-Before you can run the Secured Booster, you must update your {OpenShiftLocal} from the default configuration. You only need to do this once for the secured booster missions.
+Before you can run the Secured Booster, you must update your {OpenShiftLocal} from the default configuration. 
+You only need to do this once for any of the secured mission boosters.
+
```
Comment (Ladicek) [inline docs/topics/secured-booster-minishift.adoc]: I guess this empty line will make a paragraph break? That's probably not what we want. Otherwise LGTM.

**Comment by rhoads-zach** [comment]
thank you for the feedback @Ladicek.

Fixed the line break.

Merging.

**Pull request merged**